nrpy/infrastructures/BHaH/Makefile_helpers.py

```diff
@@ -30,6 +30,7 @@ def output_CFunctions_function_prototypes_and_construct_Makefile(
     lib_function_prefix: str = "",
     include_dirs: Optional[List[str]] = None,
     clang_format_options: str = "-style={BasedOnStyle: LLVM, ColumnLimit: 150}",
+    code_ext: str = "c",
```

Comment:
I think `src_code_file_ext` would be a clearer name. Also please put it before clang_format_options, for consistency with other functions that emit C code. Finally, since we want to support C++ properly, wouldn't it be a good idea to rename addl_CFLAGS to something like addl_compiler_flags to be language-agnostic?

Reply:
done